src/xGadgetron/cGadgetron/tests/CMakeLists.txt

```diff
@@ -35,30 +35,35 @@ if (DOWNLOAD_ZENODO_TEST_DATA)
                 "a692ba3a6c456d1ad49750717a9d5dda")
 endif()
 
-OPTION(RERUN_ISMRMRD_SHEPP_LOGAN "Generate testdata compatible with ISMRMRD Version" OFF)
+OPTION(RUN_ISMRMRD_SHEPP_LOGAN "Generate testdata compatible with ISMRMRD Version" ON)
 find_program(EXE_ISMRMRD_SL NAMES ismrmrd_generate_cartesian_shepp_logan
                 NAMES_PER_DIR
                 PATHS ${CMAKE_INSTALL_PREFIX})
 
-
-set(TESTDATA_OUTPUT_PATH "${SIRF_SOURCE_DIR}/data/examples/MR/")
-set(TESTDATA_OUTPUT_NAME_FULL "simulated_MR_2D_cartesian.h5")
-set(TESTDATA_OUTPUT_NAME_GRAPPA "simulated_MR_2D_cartesian_Grappa2.h5")
+set(TESTDATA_OUTPUT_PATH "${CMAKE_CURRENT_BINARY_DIR}/data/examples/MR")
+set(TESTDATA_OUTPUT_NAME_FULL "autogenerated_simulated_MR_2D_cartesian.h5")
+set(TESTDATA_OUTPUT_NAME_GRAPPA "autogenerated_simulated_MR_2D_cartesian_Grappa2.h5")
 
 set(TESTDATA_TEMP_PREFIX "testdata_tmp")
 
-if(EXE_ISMRMRD_SL AND RERUN_ISMRMRD_SHEPP_LOGAN)
+if(RUN_ISMRMRD_SHEPP_LOGAN)
+  if(EXE_ISMRMRD_SL)
+    if (NOT EXISTS ${TESTDATA_OUTPUT_PATH})
+      file(MAKE_DIRECTORY ${TESTDATA_OUTPUT_PATH})
+    endif()
     add_custom_target(GENERATE_SIMULATED_TESTDATA ALL
-        COMMAND rm -f ${TESTDATA_OUTPUT_NAME_FULL}
-        COMMAND rm -f ${TESTDATA_OUTPUT_NAME_GRAPPA}
-        COMMAND env LD_LIBRARY_PATH=${CMAKE_INSTALL_PREFIX}/lib:$LD_LIBRARY_PATH ${EXE_ISMRMRD_SL} -r 2 -o "${TESTDATA_TEMP_PREFIX}_1.h5"
-        COMMAND env LD_LIBRARY_PATH=${CMAKE_INSTALL_PREFIX}/lib:$LD_LIBRARY_PATH ${EXE_ISMRMRD_SL} -a 2 -o "${TESTDATA_TEMP_PREFIX}_2.h5"
+        COMMAND ${CMAKE_COMMAND} -E rm -f "${TESTDATA_OUTPUT_NAME_FULL}"
+        COMMAND ${CMAKE_COMMAND} -E rm -f "${TESTDATA_OUTPUT_NAME_GRAPPA}"
+        COMMAND env LD_LIBRARY_PATH=${CMAKE_INSTALL_PREFIX}/lib:$ENV{LD_LIBRARY_PATH} ${EXE_ISMRMRD_SL} -r 2 -o "${TESTDATA_TEMP_PREFIX}_1.h5"
+        COMMAND env LD_LIBRARY_PATH=${CMAKE_INSTALL_PREFIX}/lib:$ENV{LD_LIBRARY_PATH} ${EXE_ISMRMRD_SL} -w 64 -a 2 -o "${TESTDATA_TEMP_PREFIX}_2.h5"
```

Comment:
@paskino you said this could be done with CMake. Ideally also use `DYLD_LIBRARY_PATH` for macos, although we could skip that. the important thing is that it doesn't break windows (which doesn't have `env`).